zerver/lib/email_notifications.py

```diff
@@ -408,8 +408,15 @@ def do_send_missedmessage_events_reply_in_zulip(
     triggers = [message["trigger"] for message in missed_messages]
     unique_triggers = set(triggers)
 
+    personal_mentioned = any(
+        message["trigger"] == "mentioned" and message["mentioned_user_group_id"] is None
+        for message in missed_messages
+    )
+
     context.update(
         mention="mentioned" in unique_triggers or "wildcard_mentioned" in unique_triggers,
```

Comment:
Would it be correct to remove `wildcard_mentioned` in this condition given the other changes?

Reply:
``wildcard_mentioned`` is also treated as `mention`,  so I think removing it might break the logic.

`mention` is still being used in email subject. It could be removed but require more changes. 
https://github.com/zulip/zulip/blob/842da9e10a292c71b83b07a172d53f1a2c6bb819/templates/zerver/emails/missed_message.subject.txt#L4

Reply:
OK; feel free to play with whether you can remove it in a way that makes the codebase more readable, but I don't think it's essential given that.

Reply:
`mention` is also used in another places as well. So I think just leave it as it is.
https://github.com/zulip/zulip/blob/544b88e7ecb6dceec770be60776d3f1840ad76ac/zerver/lib/email_notifications.py#L465-L468

Reply:
Sounds good!